observe/resource_monitor_v2_action_test.go

```
@@ -183,3 +191,135 @@ func TestAccObserveMonitorV2ActionWebhook(t *testing.T) {
 		},
 	})
 }
+
+func TestAccObserveMonitorV2MultipleActionsEmail(t *testing.T) {
+	randomPrefix := acctest.RandomWithPrefix("tf")
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: fmt.Sprintf(monitorV2ConfigPreamble+`
+					resource "observe_monitor_v2" "first" {
+						workspace = data.observe_workspace.default.oid
+						rule_kind = "count"
+						name = "%[1]s"
+						lookback_time = "30m"
+						comment = "a descriptive comment"
+						inputs = {
+							"test" = observe_datastream.test.dataset
+						}
+						stage {
+							pipeline = <<-EOF
+								colmake kind:"test", description:"test"
+							EOF
+							output_stage = true
+						}
+						stage {
+							pipeline = <<-EOF
+								filter kind ~ "test"
+							EOF
+						}
+						rules {
+							level = "informational"
+							count {
+								compare_values {
+									compare_fn = "greater"
+									value_int64 = [0]
+								}
+							}
+						}
+						scheduling {
+							interval {
+								interval = "15m"
+								randomize = "0"
+							}
+						}
+						actions {
+							oid = observe_monitor_v2_action.act1.oid
+							levels = ["critical"]
```

Interesting, does this work? We only have an informational rule amongst the rules block.
I would think the api should error out if we refer to an unknown severity?

---

The test in question passed. I'm pretty sure the answer here is that the API allows triggers to be defined even if they aren't accessible via the current rules.

---

I have also produced this locally without issues

---

But for the sake of having the test make sense in mapping to a real use-case, I'll change this to informational